<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D-Audio 2V Doom</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(10, 15, 24);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="displayLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>799</width>
      <height>419</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="dumpButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dump</string>
    </property>
   </widget>
   <widget class="QLabel" name="buckleInLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>799</width>
      <height>419</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>55</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="DoomResources.qrc">:/images/doom-nightmare-buckle-in.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>displayLabel</zorder>
   <zorder>buckleInLabel</zorder>
   <zorder>dumpButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="DoomResources.qrc"/>
 </resources>
 <connections/>
</ui>
